src/pages/video-test.tsx

```diff
@@ -40,9 +40,10 @@ const EggheadPlayer: React.FC<{
   notesUrl: string
 }> = ({videoResource, notesUrl}) => {
   const playerContainer = React.useRef<any>()
-  const playerPrefs = useEggheadPlayerPrefs()
+  const {setPlayerPrefs, getPlayerPrefs} = useEggheadPlayerPrefs()
   const {player} = usePlayer()
   const {activeMetadataTracks = []} = player
+  const {activeSidebarTab} = getPlayerPrefs()
```

Comment:
was having a difficult time getting this to update from the prefs hook until I made a getter which now reads at ender and updates

I thought calling `set` on a `useState` would trigger the update but for some reason we aren't getting a refreshed value even though the underlying state is update.